src/main/java/maksym/fedorenko/bookstore/model/OrderItem.java

```diff
@@ -0,0 +1,49 @@
+package maksym.fedorenko.bookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
+
+@Entity
+@Table(name = "order_items")
+@SQLDelete(sql = "UPDATE order_items SET is_deleted = true WHERE id = ?")
+@Where(clause = "is_deleted=false")
```

Comment:
do we really need soft delete for order item? why not just deleting it completely?

Reply:
I thought that we need to save deleted orders and order items for maybe some statistics or something else. I agree that information about cart items is redundant, but we may need information about the item order in the future.

Reply:
well, for the sake of auditing...